src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -347,35 +348,59 @@ public void visitToken(DetailAST ast) {
         if (!allowMultipleEmptyLinesInsideClassMembers) {
             processMultipleLinesInside(ast);
         }
-
+        if (ast.getType() == TokenTypes.PACKAGE_DEF) {
+            checkCommentInModifiers(ast);
```

Comment:
Is this change directed at only package definitions?

https://strkkk.github.io/checkstyle/4763/checkstyle/index.html#A11
https://strkkk.github.io/checkstyle/4763/checkstyle/xref/checkstyle/contribution/checkstyle-tester/repositories/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationCustomAnnotation.java.html#L101
This is not a package definition that is showing differences.

Reply:
This is false positive.
Good catch.

Reply:
It is probably caused by first commit, which supposed to be a refactoring.
I will remove it and rerun report

Reply:
So I am correct that this change should only affect package definitions?

PR title says package, but issue title doesn't mention package even though package is the only example in the issue.

Reply:
Yes, it is targeted only for package definition

Reply:
This regression was caused by refactoring commit. I removed it, will let you know when new report will be uploaded.

Reply:
@rnveach I removed commit and updated report, please see first post